tests/cpp/aot_metadata_test.cc

```diff
@@ -39,9 +40,19 @@ const struct TVMTensorInfo kNormalOutputs[1] = {
 
 const int64_t kNormalPool1Shape[3] = {3, 8, 8};
 const struct TVMTensorInfo kNormalPools[1] = {{"pool1", kNormalPool1Shape, 3, DLDataType{3, 4, 7}}};
+const struct TVMConstantInfo kNormalConsts[1] = {{"consts1", 0, 0, {}}};
 
 const struct TVMMetadata kNormal = {
-    TVM_METADATA_VERSION, kNormalInputs, 2, kNormalOutputs, 1, kNormalPools, 1, "default",
+    TVM_METADATA_VERSION,
+    kNormalInputs,
+    2,
+    kNormalOutputs,
+    1,
+    kNormalPools,
+    1,
+    kNormalConsts,
```

Comment:
kNormalConstantPools might be better here.

Reply:
amended